_docs/samples/bookinfo.md

```diff
@@ -162,6 +162,49 @@ This application is polyglot, i.e., the microservices are written in different l
    200
    ```
 
+1. If you enabled auth and want to play with it, you can use curl from one envoy to send request to other services. For example, you want to ssh into the envoy container of details service, and send request to other services by curl. There are several steps:
+   
+   Step 1: get the details pod name
+   ```bash
+   kubectl get pods | grep details 
+   ```
+   ```bash
+   details-v1-4184313719-5mxjc       2/2       Running   0          18h
+   ```
+
+   Make sure the pod is "Running".
+
+   Step 2: ssh into the envoy container 
+   ```bash
+   kubectl exec -it details-v1-4184313719-5mxjc -c proxy /bin/bash 
+   ```
+
+   Step 3: make sure the key/cert is in /etc/certs/ directory
+   ```bash
+   ls /etc/certs/ 
+   ````
+   ```bash
+   cert-chain.pem   key.pem 
+   ```` 
+   
+   Step 4: send requests to another service, for example, productpage.
+   ```bash
+   curl https://productpage:9080 -v --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem -k
```

Comment:
-k is for client not to check server cert,

--key is to present client to server